src/commands-extender/print-preview.component.html

```diff
@@ -1,4 +1,4 @@
 <div style="position: fixed;top: 50%;left: 50%;">
-    <h1 *ngIf="dataItem">{{dataItem.Title}}</h1>
+    <h1 *ngIf="dataItem" [textContent]="dataItem.Title"></h1>
```

Comment:
check whether this could be reverted to the previous code

Reply:
What Stefan said applies. It is visualized in the UI as {{dataItem.Title}} instead of the title of the item

![image](https://user-images.githubusercontent.com/19408643/133579435-6072491f-6fdf-416f-be0c-32c4dc9f66b9.png)